include/i386/libk++/iostream.h

```diff
@@ -24,7 +24,7 @@
 
 #define klog(...) ({                                                         \
     ((firefly::libkern::get_cursor_handle() << __FUNCTION__                  \
-                                            << ": " << __VA_ARGS__ << ' ')); \
+                                            << ": " << __VA_ARGS__)); \
```

Comment:
Change klog() to this:

```cpp
#define klog() firefly::libkern::get_cursor_handle() << __FUNCTION__ << ":" << __LINE__ << ": "
```

Then you can just
```cpp
klog() << "Something went wrong";
```

Reply:
The only printing routine that has explicit support for numerical data types is `printf`
Want me to reformat the macro to use itoa/libkern::dec?

Reply:
Does `cursor` not allow integers? 🤔

Reply:
Ah no hang on I know what I did there.
A size_t was used to override the `<<` operator instead of an integer to allow for greater flexibility with few LoC.
Casting `__LINE__` to a size_t will fix the problem

Reply:
sure